whatsapp_invoice.php

```diff
@@ -0,0 +1,185 @@
+<?php
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    $to = $_POST['to'];
+    $message = $_POST['message'];
+    $file = $_FILES['file'];
+
+    // Specify the uploads folder i have made in root
+    $uploadDir = __DIR__ . '/upload/';
+
+    // Create the uploads directory 
+    if (!file_exists($uploadDir)) {
+        mkdir($uploadDir, 0777, true);
+    }
+
+    // Move the uploaded file to the local storage
+    $filename = uniqid() . '_' . $file['name'];
+    $filePath = $uploadDir . $filename;
+    move_uploaded_file($file['tmp_name'], $filePath);
+
+    // Store the timestamp when the file was uploaded
+    $timestampFile = $uploadDir . 'timestamps/' . $filename . '.txt';
+    file_put_contents($timestampFile, time());
+
+    // Include the link to the file in the message
+    // $fileLink = "http://localhost:3000/uploads/$filename";
+    $fileLink = "https://www.africau.edu/images/default/sample.pdf";
```

Comment:
Need to generate dynamic invoice.pdf , store it. this create a URL, then add settimeout to this file for 30 min.

Reply:
I have used set time out function in the cleanup file